hadoop-tools/hadoop-fs2img/src/main/java/org/apache/hadoop/hdfs/server/namenode/TreePath.java

```diff
@@ -141,7 +158,11 @@ INode toFile(UGIResolver ugi, BlockResolver blk,
             "Exact path handle not supported by filesystem " + fs.toString());
       }
     }
-    // TODO: storage policy should be configurable per path; use BlockResolver
+    if (aclStatus != null) {
+      throw new UnsupportedOperationException(
+          "Acls not supported by ImageWriter");
+    }
+    //TODO: storage policy should be configurable per path; use BlockResolver
```

Comment:
I usually prefer to leave TODO in smaller functions. So I would extract whatever is needed.

Reply:
This is an existing comment, will need to build some context to improve it.